fk/secretsend_test.go

```diff
@@ -47,6 +48,81 @@ func TestEncryptSecret(t *testing.T) {
 
 }
 
+type mockReadFile struct {
+	readFileBytes []byte
+	readFileError error
+}
+
+func (m mockReadFile) ReadFile(filename string) ([]byte, error) {
+	return m.readFileBytes, m.readFileError
+}
+
+type mockYesNoPrompter struct {
+	answer bool
+}
+
+func (m mockYesNoPrompter) promptYesNo(question string, defaultAnswer string, key *pgpkey.PgpKey) bool {
+	return m.answer
+}
+
+func TestGetSecretFromFile(t *testing.T) {
+	t.Run("returns the file contents as string", func(t *testing.T) {
+		fileReader := mockReadFile{
+			readFileError: nil,
+			readFileBytes: []byte("hello"),
+		}
+
+		prompter := mockYesNoPrompter{
+			answer: true,
+		}
+
+		secret, err := getSecretFromFile("/fake/filename", fileReader, prompter)
+		assert.ErrorIsNil(t, err)
+		assert.Equal(t, "hello", secret)
+	})
+
+	t.Run("works with fileReader=nil", func(t *testing.T) {
```

Comment:
Are you missing a test here? Or is this now redundant?

Reply:
oops. fixed (fixed up)